ocs_ci/ocs/bucket_utils.py

```diff
@@ -2023,3 +2042,46 @@ def sample_if_objects_expired(mcg_obj, bucket_name, prefix="", timeout=600, slee
 
     assert sampler.wait_for_func_status(result=True), f"{message} are not expired"
     logger.info(f"{message} are expired")
+
+def upload_test_objects_to_source_and_wait_for_replication(
+    mcg_obj, source_bucket, target_bucket, mockup_logger, timeout
+):
+    """
+    Upload a set of objects to the source bucket, logs the operations and wait for the replication to complete.
+
+    """
+    logger.info("Uploading test objects and waiting for replication to complete")
+    mockup_logger.upload_test_objs_and_log(source_bucket.name)
+
+    logger.info(
+        "Resetting the noobaa-core pod to trigger the replication background worker"
+    )
```

Comment:
Since the reset of the pod doesn't happen after this log, this is probably a leftover log that should also be removed at ocs-ci/tests/manage/mcg/test_log_based_bucket_replication.py :)